scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -306,7 +306,7 @@ monitor_sst_progress() {
       break;
     fi
 
-    current_size=$(du --max-depth=1 ${tmpsstdir} | tail -n 1 | awk '{print $1}')
+    current_size=$(du -b -s ${tmpsstdir} | tail -n 1 | awk '{print $1}')
```

Comment:
do we need `| tail -1` if we use --summarize and only ${tmpsstdir} as target ?

Reply:
Removed `| tail -n 1`